sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -872,31 +872,21 @@ object StringTrim {
     Examples:
       > SELECT _FUNC_('    SparkSQL   ');
        SparkSQL
-      > SELECT _FUNC_(BOTH FROM '    SparkSQL   ');
```

Comment:
it's a bit bad to drop these examples.

Reply:
Maybe we should think about a better way to add function alias, so that they can have their own examples.